novatel_gps_driver/CMakeLists.txt

```diff
@@ -125,3 +129,5 @@ install(FILES nodelet_plugins.xml
 
 ### Install Launch Files ###
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+add_dependencies(${PROJECT_NAME} ${catkin_EXPORTED_TARGETS})
```

Comment:
Move this line to line 76 so it is near the executable it references to make reading this file clearer.

Reply:
It was put there by swri robotics. Anyway moved it.